console/apps/webapp/public/config.js

```
@@ -18,28 +18,28 @@
       idToken: {
-        // Disable for Thunder / local dev with non-standard issuers or self-signed certs
-        validate: false,
-        clockTolerance: 300,
+        validate: '' === 'true',
+        clockTolerance: Number('') || 300,
       },
     },
     storage: 'localStorage',
   },
-  disableAuth: true,
+  disableAuth: 'true' === 'true',
```

Biome (2.4.13)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)